changelog.d/11895.removal

```diff
@@ -0,0 +1 @@
+Drop support for `webclient` listeners, they are now ignored. The `web_client_location` configuration must point to a HTTP(S) URL, if configured.
```

Comment:
> they are now ignored

It looks like they're now raising errors rather than being ignored?

Reply:
Good catch, thank you!